File: problems/hackerrank/Tutorials/30 Days of Code/Day 2: Operators/solution.rb
# Take inputs
meal_cost = gets.to_f
tip_percent = gets.to_f
tax_percent = gets.to_f

# Calculate
tip = meal_cost * tip_percent / 100
tax = meal_cost * tax_percent / 100
total = (meal_cost + tip + tax).round

# Print
puts "The total meal cost is #{total} dollars."

File: problems/hackerrank/Tutorials/30 Days of Code/Day 2: Operators/solution.py
# Take inputs
meal_cost = float(input())
tip_percent = float(input())
tax_percent = float(input())

# Calculate
tip = meal_cost * tip_percent / 100
tax = meal_cost * tax_percent / 100
total = round(meal_cost + tip + tax)

# Print
print("The total meal cost is {} dollars.".format(total))
